server/base/src/main/java/org/apache/accumulo/server/util/ListVolumesUsed.java

```diff
@@ -131,7 +113,11 @@ private static void listTable(Ample.DataLevel level, ServerContext context) thro
   }
 
   public static void listVolumes(ServerContext context) throws Exception {
-    listZookeeper(context);
+    try {
+      listTable(Ample.DataLevel.ROOT, context);
+    } catch (UnsupportedOperationException ex) {
+      // print nothing if no table name exists
+    }
```

Comment:
Why are you suppressing the error here?  You could just let the error be thrown, like the other calls to ```listTable```

Reply:
If it's not caught, the command errors out without completing. The previous method just iterated over a collection of 0 elements and continued processing.

Reply:
The other option would be to print out that there are no volumes in ZK to display. But the previous method did not do so and I was attempting to mimic its behavior.

Reply:
I see.  Yeah I would at least print there are no volumes, that would be helpful.